docker-compose.yml

```
@@ -30,16 +30,16 @@ services:
       ME_CONFIG_MONGODB_URL: "mongodb://${MONGODB_ROOT_USERNAME}:${MONGODB_ROOT_PASSWORD}@mongo:27017/"
   
   minio:
-    image: minio/minio:RELEASE.2025-02-07T23-21-09Z
+    image: minio/minio:RELEASE.2025-07-23T15-54-02Z
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)